ranger/config/commands.py

```diff
@@ -1990,3 +1990,34 @@ def make_safe_path(dst):
 
     def execute(self):
         return self.fm.paste(make_safe_path=paste_ext.make_safe_path)
+
+
+class confirm(Command):
+    """
+    :confirm <command> [ARGS...]
+
+    Can be used with any command. Ranger will ask if it should execute the
+    given command.
+    """
+
+    def execute(self):
+        if len(self.args) <= 1:
+            self.fm.notify("Nothing to confirm.")
+        elif self.arg(1) == 'chain':
+            # TODO: This doesn't work for chained chain commands.
+            for command in [arg.lstrip() for arg in self.rest(2).split(';')]:
+                self._ask_for_confirmation(command)
+        else:
+            self._ask_for_confirmation(self.rest(1))
+
+    def _question_callback(self, command, answer):
+        if answer in ('y', 'Y'):
```

Comment:
Let's use `answer.lower()` here, this way there will be no need to enumerate every variant by hand.